storage/google/cloud/storage/acl.py

```diff
@@ -196,7 +196,7 @@ class ACL(object):
         'bucketOwnerFullControl',
     ])
     """See:
-    https://cloud.google.com/storage/docs/access-control#predefined-acl
+    https://cloud.google.com/storage/docs/access-control/
```

Comment:
Ah thanks! The tool I'm using shows broken anchors as well so that's why it came up. My fault for not finding the alternative link.